src/lib/components/QuillEditor/QuillToolbar.tsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import './editor.styles.css';
+
+export interface IQuillToolbarProps {}
+
+// interface IExtraQuillField extends Quill {
+//   whitelist: string[];
+// }
+
+// const Font = Quill.import('formats/font') as IExtraQuillField;
+// const Size = Quill.import('formats/size') as IExtraQuillField;
+
+// Font.whitelist = [
+//   'arial',
+//   'comic-sans',
+//   'courier-new',
+//   'georgia',
+//   'helvetica',
+//   'lucida',
+// ];
+
+// Size.whitelist = ['extra-small', 'small', 'medium', 'large'];
+
+// Quill.register(Size, true);
+// Quill.register(Font, true);
+
+// function imageHandler() {
+//     const range = Quill.getSelection();
+//     const value = prompt('What is the image URL');
+//     if(value) {
+//       Quill.insertEmbed(range.index, 'image', value, Quill.sources.USER);
+//     }
+// };
+
+const QuillToolbar: React.FC<IQuillToolbarProps> = () => {
+  return (
+    <div id="toolbar">
+      <span className="ql-formats">
+        <select className="ql-font">
+          <option value="arial">Arial</option>
+          {/* <option value="comic-sans">Comic Sans</option>
+          <option value="courier-new">Courier New</option>
+          <option value="georgia">Georgia</option>
+          <option value="helvetica">Helvetica</option>
+          <option value="lucida">Lucida</option> */}
+        </select>
+        <select className="ql-size">
+          {/* <option value="extra-small">Size 1</option>
+          <option value="small">Size 2</option> */}
+          <option value="medium">Size 1</option>
+          {/* <option value="large">Size 4</option> */}
+        </select>
+        <select className="ql-header">
+          <option value="1">Заголовок 1</option>
+          <option value="2">Заголовок 2</option>
+          <option selected />
+          {/* <option value="3">Звичайний</option> */}
+        </select>
```

Comment:
```suggestion
        <select
          className="ql-header"
          defaultValue=""
          onBlur={(e) => e.persist()}
        >
          <option value="1">Заголовок 1</option>
          <option value="2">Заголовок 2</option>
          <option value="" selected>
            Звичайний
          </option>
        </select>
```

Tru this solution, It should fix the problem.

Reply:
Done
Thanks!